src/chart/generateCategoricalChart.js

```diff
@@ -44,6 +44,7 @@ const generateCategoricalChart = (ChartComponent, GraphicalChild) => {
     static propTypes = {
       ...ChartComponent.propTypes,
       syncId: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+      compact: PropTypes.bool,
```

Comment:
> As #205 requested, exporting to all-in-one SVG, requires either:
> 
> - Implement Legend with pure SVG, need to calculate text's width and handle potential line wrapping, or:

As of e597e3b, Chart's `compact=true` may also be used as a foundation of "all-in-one SVG", once pure SVG Legend is supported, #205 could be resolved.